dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/WSKeyStore.java

```diff
@@ -1576,6 +1614,26 @@ private void setCertificateEntryNoStore(String alias, Certificate cert) throws K
         if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled()) {
             Tr.entry(this, tc, "setCertificateEntryNoStore", new Object[] { alias, cert });
         }
+        try {
```

Comment:
Delete the JVM dump call code. Make sure you keep the acquireWriteLock() call.

Reply:
Fixed